producer-kafka/pom.xml

```diff
@@ -31,12 +31,12 @@
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-api</artifactId>
-      <version>1.7.30</version>
     </dependency>
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>log4j-over-slf4j</artifactId>
       <version>1.7.30</version>
+      <scope>runtime</scope>
```

Comment:
not related, but as we discussed - all extensions should have slf4j provided at runtime